gettingStarted/installation/installation.html

```diff
@@ -0,0 +1,188 @@
+                                <a href="/components/Avatar/avatar.html" class="siderbar-btn">Avatar</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Badge/badge.html" class="siderbar-btn">Badge</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Button/button.html" class="siderbar-btn">Button</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Card/card.html" class="siderbar-btn">Card</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Image/image.html" class="siderbar-btn">Image</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Input/input.html" class="siderbar-btn">Input</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/List/list.html" class="siderbar-btn">List</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Navigation/navigation.html" class="siderbar-btn">Navigation</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Modal/modal.html" class="siderbar-btn">Modal</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Rating/rating.html" class="siderbar-btn">Rating</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Slider/slider.html" class="siderbar-btn">Slider</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Grid/grid.html" class="siderbar-btn">Grid</a>
+                            </li>
+                            <li class="sidebar-btn-content">
+                                <a href="/components/Toast/toast.html" class="siderbar-btn">Toast</a>
+                            </li>
+                        </ul>
+                    </ul>
+                </ui>
+            </div>
+            <div class="main-component-container">
+                <div class="component-title">Installation</div>
+                <div class="component-desc">
+                    In order to get started with NovaUI within your project, Copy the Link given below and paste it in
```

Comment:
Your project name is EssenceUI. Here it says NovaUI?

Reply:
in the beginning i had thought to name it nvoa but then i changed the name

Reply:
Okay